tests/python/contrib/test_vsi_npu/test_vsi_tflite_model_all.py

```diff
@@ -199,53 +203,119 @@ def get_model(model_list, model_name):
      'dtype': "uint8"},
 ]
 
-model_full_name = os.environ["TFLITE_MODEL"]
-(_, model_name) = os.path.split(model_full_name)
+def process(model_name):
+    model = get_model(model_list, model_name)
+    logging.info(model)
+    shape = model['shape']
+    input_tensor_name = model['input_tensor_name']
+    DTYPE = model['dtype']
+    # wait=input("press any key and continue...")
+
+    path = "./"
+    img = Image.open(path + "space_shuttle_224x224.jpg")
+    img = img.resize((shape[1], shape[2]))
+    n1 = np.array(img)
+    #n1 = n1[:, :, 0] # pick one channel
+    #n1 = np.broadcast_to(n1, (4, 224, 224, 3)) # batch the image
+    n1 = n1.reshape(shape)
+    input_data = n1.astype(np.uint8)
+
+    # input_data = np.ones(shape, DTYPE)
+
+    vsi_input_data = {
+        input_tensor_name: tvm.nd.array(input_data),
+    }
+    ref_output = get_ref_result(shape, model_name, input_data, input_tensor_name, DTYPE)
+    vsi_output = compile_tflite_model(shape, model_name, vsi_input_data, input_tensor_name, DTYPE)
+
+    #logging.info("ref_output:{}".format(ref_output))
+    #logging.info("vsi_output:{}".format(vsi_output))
+
+    if DTYPE == "uint8":
+        tolerance = 5
+    else:
+        tolerance = 1e-3
+
+    result = abs(vsi_output.astype("float32") - ref_output.astype("float32"))
+    result0 = result < tolerance
```

Comment:
any method from pytest to check the error between CPU & NPU ? 
We should not compare results by ourselves.